WreckEngine/Text.cpp

```diff
@@ -1,198 +1,549 @@
 #include "Text.h"
-
+#include "ShaderHelper.h"
+#include "External.h"
+#include FT_ERRORS_H
+#include FT_GLYPH_H
+#include FT_OUTLINE_H
+#include FT_BITMAP_H
+#include <array>
 #include <iostream>
 
-#include "External.h"
+#define STB_RECT_PACK_IMPLEMENTATION
+#include <stb_rect_pack.h>
+
+namespace
+{
+    struct FT_Wrapper
```

Comment:
Because it's unnecessary in the modern age! With the advent and rising popularity of 1920x1080 as a standard resolution, there's no need to make code one line anymore! Same with having curly braces on the same line! You have enough screen space to put them on new lines nowadays!